libcontainer/cgroups/systemd/v2.go

```diff
@@ -337,12 +338,12 @@ func (m *unifiedManager) getSliceFull() (string, error) {
 	}
 
 	if m.rootless {
-		dbusConnection, err := m.dbus.getConnection()
-		if err != nil {
-			return "", err
-		}
-		// managerCGQuoted is typically "/user.slice/user-${uid}.slice/user@${uid}.service" including the quote symbols
```

Comment:
Can we retain this comment

Reply:
right; fixed

Reply:
Doesn’t seem fixed

Reply:
`ENOTENOUGHCOFFEE`

Let me try again :^)

Reply:
Yay, it worked this time!